src/GifWin/GifWitConversionProgressWindow.xaml.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Windows;
+using GifWin.ViewModels;
+
+namespace GifWin
+{
+    /// <summary>
+    /// Interaction logic for GifWitConversionProgressWindow.xaml
+    /// </summary>
+    public partial class GifWitConversionProgressWindow : Window
+    {
+        public GifWitConversionProgressWindow()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnContentRendered(EventArgs e)
+        {
+            ((GifWitConversionProgressViewModel)DataContext).ConvertLibrary();
```

Comment:
Hmm, how would that work? I want the task automatically kicked off after the window shows, but the data context changes _before_?